snmp/tests/compose/data/idrac.snmprec

```diff
@@ -1,4 +1,4 @@
-1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.674.10892.1.2
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.674.10892.2.2
```

Comment:
This changes it to something that doesn't match the power edge profile, as we load them all now.